trade-finance-manager-api/src/v1/emails/MIA-confirmation/generate-email-variables.api-test.js

```diff
@@ -0,0 +1,64 @@
+const {
+  generateMiaConfirmationEmailVars,
+  commonEmailVars,
+} = require('./generate-email-variables');
+const mapSubmittedDeal = require('../../mappings/map-submitted-deal');
+const bssEmailVariables = require('./bss-email-variables');
+const CONSTANTS = require('../../../constants');
+const MOCK_BSS_DEAL = require('../../__mocks__/mock-deal');
+const MOCK_GEF_DEAL = require('../../__mocks__/mock-gef-deal');
+
+describe('generate AIN/MIN confirmation email variables', () => {
+  const mockFacilityLists = {};
+
+  describe('commonEmailVars', () => {
+    it('should return object of email variables used in both BSS and GEF', () => {
+      const mockDeal = mapSubmittedDeal({ dealSnapshot: MOCK_BSS_DEAL });
+
+      const result = commonEmailVars(mockDeal);
+
+      const expected = {
+        recipientName: `${mockDeal.maker.firstname} ${mockDeal.maker.surname}`,
+        exporterName: mockDeal.exporter.companyName,
+        bankReferenceNumber: mockDeal.bankReferenceNumber,
+        ukefDealId: mockDeal.ukefDealId,
+      };
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe(`when dealType is ${CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS}`, () => {
+    const mockDeal = mapSubmittedDeal({ dealSnapshot: MOCK_BSS_DEAL });
```

Comment:
this is really minor: do we need this line here? or we can include it inside the `it` statement below

Reply:
Any feedback is good feedback 😉 I see what you mean - I done this because if the scenario/describe needs to be elaborated, it's already got the core deal required, if that makes sense? Happy to move to the `it` statement though.

I'll merge this, can do another PR after if you like? Interested what you think

Reply:
copy that.
we can leave it as it is. most likely it will get expanded at some point